lib/components/narrative/trip-tools.js

```diff
@@ -169,91 +170,102 @@ LinkButton.propTypes = {
   url: PropTypes.string
 }
 
-class TripTools extends Component {
-  static defaultProps = {
-    buttonTypes: [
-      'COPY_URL',
-      'PRINT',
-      'REPORT_ISSUE',
-      'START_OVER',
-      'POPUP_LINK'
-    ]
-  }
-
-  render() {
-    const {
-      buttonTypes,
-      location,
-      popupTarget,
-      reactRouterConfig,
-      reportConfig,
-      setPopupContent
-    } = this.props
+const TripTools = ({
+  buttonTypes,
+  intl,
+  location,
+  popupTarget,
+  reactRouterConfig,
+  reportConfig,
+  setPopupContent
+}) => {
+  const { SvgIcon } = useContext(ComponentContext)
+  const PopupIcon = useMemo(() => {
+    const IconComponent = (componentProps) => {
+      return (
+        <StyledIconBase>
+          <SvgIcon iconName={popupTarget} {...componentProps} />
+        </StyledIconBase>
+      )
+    }
+    return IconComponent
+  }, [SvgIcon, popupTarget])
 
-    const buttonComponents = []
-    buttonTypes.forEach((type) => {
-      switch (type) {
-        case 'COPY_URL':
-          buttonComponents.push(<CopyUrlButton />)
-          break
-        case 'PRINT':
-          buttonComponents.push(<PrintButton />)
-          break
-        case 'REPORT_ISSUE':
-          if (!reportConfig || !reportConfig.mailto) break
-          buttonComponents.push(<ReportIssueButton {...reportConfig} />)
-          break
-        case 'START_OVER':
-          const startOverUrl = startOver(
-            reactRouterConfig?.basename,
-            location?.search
-          )
+  const buttonComponents = []
+  buttonTypes.forEach((type) => {
+    switch (type) {
+      case 'COPY_URL':
+        buttonComponents.push(<CopyUrlButton />)
+        break
+      case 'PRINT':
+        buttonComponents.push(<PrintButton />)
+        break
+      case 'REPORT_ISSUE':
+        if (!reportConfig || !reportConfig.mailto) break
+        buttonComponents.push(<ReportIssueButton {...reportConfig} />)
```

Comment:
Could the extra break be removed if we flipped the if condition?

Reply:
Fixed in https://github.com/opentripplanner/otp-react-redux/pull/729/commits/866fdc2bfbe16ae5c0c97c0868f3bd3a6f37e541.